shepherd_bms/src/main.cpp

```diff
@@ -347,11 +347,12 @@ void shepherdMain()
 	// Check for faults
 	bmsFault |= faultCheck(analyzer.bmsdata);
 
-	// ACTIVE/NORMAL STATE
+	// ACTIVE/NORMAL STATE // Shove this into handlestate for active state in SM
 	if (bmsFault == FAULTS_CLEAR) 
 	{
 		compute.setFault(NOT_FAULTED);
```

Comment:
This can prob be done in two lines rather than a big if: `fault_state = bmsFault == FAULTS_CLEAR; compute.setFault(fault_state);`

Reply:
Also we probably want to move the fault check into the handleState entry point, right?

Reply:
yes this is being done